02_Intro_to_Elixir/05-operators-and-variables.markdown

```diff
@@ -3,8 +3,73 @@ layout: page
 title: Operators and Variables
 date: 2016-10-1 13:38:30 -0700
 ---
+### Comparison
+Let's take a look at how we might compare variables in Elixir
 
-# Operators
+Here are some examples.
+
+ Try typing the examples below into your console
+
+```elixir
+1 + 1 == 2
+1 > 2
+2 <= 4
+1 != 2
+```
+
+### Strict Comparison
+```elixir 
+2 == 2.0 # true
+2 === 2.0 # false
+```
+
+What is different about the expression above?
+*hint: Thing about data types. 
+
+## Variables
+
+In Elixir, we'll use variables all over the place. Assign values to variables in elixir works differently than it does in imperative languages like Ruby or Javascript.
+
+However for now let's pretend that "assignment" works just as it does in other languages. 
+
+We will go into what is happening under the hood in the section on Pattern Matching. 
+
+```elixir
+iex> x = 2
+iex> y = 3
+```
+
+What values do x and y have now?
+
+All of the following can be variables
+
+```elixir
+a = 1
+b = 0x0F
+c = div(b, a)
+```
+
+### How Assignment Actually works in Elixir
+```elixir
+iex> x = 2
+iex> y = 3
+```
+
+If we look at the code above. It looks like we have variables on the left-hand side of our expression and values on the right. And it appears that the values on the right are being "assigned" to the values on the left. However in elixir there is no assignment operator. The "=" is called the match operator. What does this mean?
```

Comment:
I think we can address any concerns in conversation with the students. The "=" is a core and tricky concept, so we may want to dwell on it awhile during the training.